include/parser.h

```diff
@@ -38,8 +29,8 @@ void init_parser(FILE *fp);
 /**
     @brief Parse file into a string array
     @param fp: pointer to the file to be read
-    @return a pointer to the struct StringArray.
+    @return a pointer to the array, setting last entry to NULL
  */
-stringArray *parse_to_arr(FILE *fp);
+char **parse_to_arr(FILE *fp);
```

Comment:
Why does this still exist given that `parse_file_buffered` exists in `search.c`?

Reply:
We may need this to parse a batch file of search terms, in which case we want the entire file instead of buffered reading...?

Reply:
Ahh, gotcha. That makes sense.